app/Models/Comment.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Comment extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $fillable = ['post_id', 'author_id', 'content'];
```

Comment:
Can be replaced with `guarded` (which is the opposite of `fillable`):
```suggestion
    protected $guarded = ['id'];
```

Reply:
oh that's great